READMEs/c2d-flow.md

```diff
@@ -238,11 +238,11 @@ ALGO_compute_input = ComputeInput(ALGO_asset, algo_service)
 datasets, algorithm = ocean.assets.pay_for_compute_service(
     datasets=[DATA_compute_input],
     algorithm_data=ALGO_compute_input,
-    consume_market_order_fee_address=bob_wallet.address,
+    consume_market_order_fee_address=ocean.web3.toChecksumAddress(bob_wallet.address),
```

Comment:
I would rather not have these all around the code and especially not in readmes, just in the models layer, when we are sending the address to the contracts.